All in on 'credit purchases are not annual'

camerondeleone opened the pull request:
This is a bit of a monster. We haven't really thought through the full implications of 'credit purchases are not annual' and the knock on effects on multi purchase commitments, but I wanted to take a stab at fleshing out what this might look like. We may find it too unfriendly to customers/procurement+finance departments?

Some things that fall out of this change, if we really lean into the concept:

- 'top-up' credit purchases don't make as much sense. None at all for single credit purchases, and less for 'multi-purchase commitments' as i'm calling them (prev. multi-year), though I've included what seemed reasonable to me.  feedback here please.
    - payment terms are now net 0 for second and third commitment amounts, since they're not aligned to annual terms, and it doesn't make sense to just immediately grant someone more credit the minute they run out and then give them net 30 (or does it?)
- all paid upfront multi year option no longer makes sense
- removed all reference to quarterly payments, since we have definitively moved away from those + it conflicts w this model.

Some obvious outstanding questions:
- credit still expires in one year (as the economics change when people amortize longer than that), but how should multi-purchase commitments be constrained? Contract language will need revision here. One year max between purchases seems sensible to me.
- Do we get rid of the concept of top-up credits entirely and just require that if people commit and burn through they purchase the original commitment amount?

camerondeleone committed:
All in on 'credit purchases are not annual'

Updated contract rules to clarify multi-purchase commitments and payment methods.

simfish85 commented on contents/handbook/growth/sales/contract-rules.md:
```diff
@@ -15,8 +15,7 @@ Although our standard monthly [pricing](/pricing) has volume discounts built in,
 2. **Prepaid credit:** We offer a 20% discount on a paid-up-front plan of >$20k, because it's beneficial to PostHog to have a customer committed.
 3. **Higher spend:** We offer an additional 5% and then 15% on top of the prepaid credit plans when the customer spends above $60k and $100k respectively.
-4. **Multi-year:** We offer an additional 5% for each additional committed credit renewal under a prepaid contract, as longer commitments are beneficial to PostHog (see table below). Credits rollover in-between for multi-year contracts.
-5. **Up-front payment:** We offer an additional 2.5% per year where a _multi-year_ contract is all **paid up-front**, as it's better for PostHog to have the money in the bank.
```
I do still want to incentivise up-front payment for longer term commitments as it's better for us to have the cash in the bank when a customer is committing to be with us for 2 or 3 years; and as such the alternative in this proposal is that they buy up front credits but the max term is 12 months.

camerondeleone committed:
clarify multi-purchase / annual interaction

Clarified terms for multi-purchase commitments and up-front payments, including conditions for discounts and credit rollover.

simfish85 reviewed (approved):
Great thanks for this - just 1 typo

simfish85 committed:
Update contents/handbook/growth/sales/contract-rules.md

charlescook-ph commented:
Is this good to merge?
